README.md

```diff
@@ -0,0 +1,150 @@
+# GLaDOS
+
+## Generic Language and Data Operand Syntax
+
+### 🔗 Useful Links
+
+- 🎨 Design & Architecture: [Figma Board](https://www.figma.com/design/nEfFaG3XVqWuiBLY3HY8iH/)
+- 📋 Project Management: [GitHub Project](https://github.com/orgs/Sigmapitech/projects/15)
+- 📚 BNF Grammar: [Railroad Diagram Generator](https://www.bottlecaps.de/rr/ui)
+- 🧪 BNF Playground: [Online Tester](https://bnfplayground.pauliankline.com/)
+
+### 📋 Overview
+
+GLaDOS is a functional programming language designed and implemented in Haskell. It features a custom LISP-like syntax (bootstrap), a stack-based virtual machine, and a compiler.
+
+**Core Components:**
+
+- **AST**: Abstract Syntax Tree definitions.
+
+- **Compiler**: Transforms source code into bytecode.
+
+- **VM**: A Stack Machine to execute the compiled bytecode.
+
+### 🚀 Getting Started
+
+We use **Nix Flakes** to ensure a reproducible development environment across all Linux distributions.
+
+**1. Prerequisites**
+[Install Nix](https://nixos.org/download/) (if you haven't already):
+
+```Shell
+# Multi-user installation
+sh <(curl --proto '=https' --tlsv1.2 -L https://nixos.org/nix/install) --daemon
+# Single-user installation
+sh <(curl --proto '=https' --tlsv1.2 -L https://nixos.org/nix/install) --no-daemon
+```
```

Comment:
Do not vendor the instruction from the official install page, since it may change in the future